packages/frontend/src/components/common/modal/ModalActionButtons.tsx

```diff
@@ -40,6 +47,22 @@ const ModalActionButtons = ({
                             });
                         }}
                     />
+                    <MenuItem
+                        role="button"
+                        icon="duplicate"
+                        text="Duplicate"
+                        onClick={(e) => {
+                            e.preventDefault();
+                            e.stopPropagation();
+                            setItemId(data.uuid);
+                            if (data.projectUuid || data.description) {
+                                onDuplicateDashboard();
+                            }
+                            if (!data.projectUuid && !data.description) {
```

Comment:
what's this description thing ? is this the best way to differenciate between dashboards and charts ? what happens if a dashboard does not have a description (it can be null) ?